---
layout: page
title: Publications
permalink: /publications/
---

<br>

### Accepted manuscripts

+ S Weaver, SD Shank, **SJ Spielman**, M Li, SV Muse, and SL Kosakovsky Pond. 2017. Datamonkey 2.0: A modern web application for characterizing selective and other evolutionary processes. *Mol. Biol. Evol.*

+ **SJ Spielman**, S Weaver, SD Shank, BR Magalas, M Li, SL Kosakovsky Pond. *In press.* Evolution of viral genomes: Interplay between selection,
recombination and other forces. In: Anisimova M. (eds) *Evolutionary Genomics. Methods in Molecular Biology (Methods and Protocols).* Humana Press. &nbsp;<a href="https://github.com/veg/evogenomics_hyphy" class="info-link"><span class="fa fa-github" style = "color:black">

### Submitted manuscripts


+ **SJ Spielman**. 2017. [phyphy: Python package for facilitating the execution and parsing of HyPhy standard analyses](http://joss.theoj.org/papers/9f65f1bc47bf8f17d773d09928a89b90). *Under Review at The Journal of Open Source Software.* &nbsp;<a href="https://github.com/sjspielman/phyphy" class="info-link"><span class="fa fa-github" style = "color:black">


+ **SJ Spielman** and SL Kosakovsky Pond. 2017. [Relative evolutionary rate inference in HyPhy with LEISR](https://doi.org/10.1101/206011). *Under Review at PeerJ*. &nbsp;<a href="https://github.com/sjspielman/leisr_validation" class="info-link"><span class="fa fa-github" style = "color:black">




### 2017

+ EL Jackson, **SJ Spielman**, and CO Wilke. 2017. [Computational prediction of the tolerance to amino-acid deletion in green-fluorescent protein](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0164905). *PLOS ONE* 12(4): e0164905.
&nbsp;<a href="https://github.com/wilkelab/eGFP_deletion_prediction" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/JacksonSpielmanWilke2016_PLOSONE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">


+ Z Kadlecova, **SJ Spielman**, D Loerke, A Mohanakrishnan, DK Reed, and SL Schmid. 2017. [Regulation of clathrin-mediated endocytosis by hierarchical allosteric activation of AP2.](http://jcb.rupress.org/content/216/1/167) *J Cell Biol* 216(1): 167--179.
&nbsp;<a href="/files/Kadlecovaetal2017_JCB.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">



### 2016

+ **SJ Spielman**, S Wan, and CO Wilke. 2016. [A comparison of one-rate and two-rate inference frameworks for site-specific *dN/dS* estimation.](http://www.genetics.org/content/early/2016/08/15/genetics.115.185264) *Genetics* 204(2): 499--511.
&nbsp;<a href="https://github.com/sjspielman/dnds_1rate_2rate" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/SpielmanWanWilke2016_Genetics.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">

+ **SJ Spielman** and CO Wilke. 2016. [Extensively parameterized mutation--selection models reliably capture site-specific selective constraint.](http://mbe.oxfordjournals.org/content/early/2016/09/08/molbev.msw171) *Mol Biol Evol* 33(11): 2990--3002.
&nbsp;<a href="https://github.com/sjspielman/mutsel_benchmark" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/SpielmanWilke2016_MBE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">

+ EL Jackson, A Shahmoradi, **SJ Spielman**, BR Jack, and CO Wilke. 2016. [Intermediate divergence levels maximize the strength of structure–sequence correlations in enzymes and viral proteins.](http://onlinelibrary.wiley.com/doi/10.1002/pro.2920/full) *Protein Sci* 25(7): 1341-1353.
&nbsp;<a href="https://github.com/wilkelab/rate_variability_variation" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/Jacksonetal2016_ProtSci.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">

+ J Echave, **SJ Spielman**, and CO Wilke. 2016. [Causes of evolutionary rate variation among protein sites.](http://dx.doi.org/10.1038/nrg.2015.18) *Nature Rev Genet* 17: 109-121.
&nbsp;<a href="/files/EchaveSpielmanWilke2016_NRG.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">


### 2015

+ **SJ Spielman** and CO Wilke. 2015. [Pyvolve: A flexible Python module for simulating sequences along phylogenies.](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0139047) *PLOS ONE* 10(9): e0139047.
&nbsp;<a href="https://github.com/sjspielman/pyvolve" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/SpielmanWilke2015_PLoSONE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">

+ AG Meyer, **SJ Spielman**, T Bedford, and CO Wilke. 2015. [Time dependence of evolutionary metrics
during the 2009 pandemic influenza virus outbreak.](http://ve.oxfordjournals.org/content/1/1/vev006)  *Virus Evolution.* 1(1):vev006-10.
&nbsp;<a href="https://github.com/wilkelab/influenza_pH1N1_timecourse" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/Meyeretal2015_VE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">


+ **SJ Spielman** and CO Wilke. 2015. [The relationship between *dN/dS* and scaled selection coefficients.](http://mbe.oxfordjournals.org/content/32/4/1097) *Mol Biol Evol* 32(4): 1097-1108.
&nbsp;<a href="http://github.com/clauswilke/Omega_Mutsel" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/SpielmanWilke2015_MBE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">


+ **SJ Spielman**, K Kumar, and CO Wilke. 2015. [Comprehensive, structurally-curated alignment and phylogeny of vertebrate biogenic amine receptors.](http://dx.doi.org/10.7717/peerj.773) *PeerJ* 3:e773.
&nbsp;<a href="http://github.com/sjspielman/amine_receptors" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/SpielmanKumarWilke2015_PeerJ.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">


### 2014


+ **SJ Spielman**, ET Dawson, and CO Wilke. 2014. [Limited utility of residue masking for positive-selection inference.](http://mbe.oxfordjournals.org/content/31/9/2496.long) *Mol Biol Evol* 31(9): 2496-2500. 2014.
&nbsp;<a href="http://github.com/sjspielman/alignment_filtering" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/SpielmanDawsonWilke2014_MBE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">


+ A Shahmoradi, DK Sydykova, **SJ Spielman**, EL Jackson, ET Dawson, AG Meyer, and CO Wilke. 2014. [Predicting evolutionary site variability from structure in viral proteins: buriedness, packing, flexibility, and design.](http://dx.doi.org/10.1007/s00239-014-9644-x) *J Mol Evol* 79:130–142. 2014.
&nbsp;<a href="https://github.com/clauswilke/structural_prediction_of_ER" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/Shahmoradietal2014_JME.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">


+ *[PREPRINT]* **SJ Spielman**\* , AG Meyer\*, and CO Wilke. 2014. [Increased evolutionary rate in the 2014 West African Ebola outbreak is due to transient polymorphism and not positive selection.](http://dx.doi.org/10.1101/011429) *bioRxiv*. \*Authors contributed equally to this work.&nbsp;&nbsp;<a href="https://github.com/wilkelab/EBOV_H1N1" class="info-link"><span class="fa fa-github" style = "color:black">


### 2013

+ MZ Tien, AG Meyer, DK Sydykova, **SJ Spielman**, and CO Wilke. 2013. [Maximum allowed solvent accessibilites of residues in proteins.](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0080635) *PLOS One* 8(11):e80635. 2013.
&nbsp;<a href="/files/Tienetal2013_PLoSONE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">


+ **SJ Spielman** and CO Wilke. 2013. [Membrane environment imposes unique selection pressures in transmembrane domains of G-protein coupled receptors.](http://link.springer.com/article/10.1007%2Fs00239-012-9538-8) *J Mol Evol* 76(3):172-182. 2013.
&nbsp;<a href="http://github.com/sjspielman/mammalian_gpcr_selection" class="info-link"><span class="fa fa-github" style = "color:black">
&nbsp;<a href="/files/SpielmanWilke2013_JME.pdf" class="info-link"><span class="fa fa-file-pdf-o" style = "color:black">
