<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Publications</title>
  <meta name="description" content="Institute for Genomics and Evolutionary Medicine">

  <link rel="stylesheet" href="/assets/main.css">
  <link rel="canonical" href="http://localhost:4000/publications/">
  <link rel="alternate" type="application/rss+xml" title="Home" href="/feed.xml">
  <link rel="stylesheet" href="/assets/prism.css">
  <script src="https://use.fontawesome.com/a672cf20b0.js"></script>
  
</head>


  <body>
    <script src="/assets/prism.js"></script>
    <header class="site-header" role="banner">

  <div class="wrapper-header">
    
    
    <a class="site-title" href="/">Home</a>
  
    
      <nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger">
          
            
            
            <a class="page-link" href="/cv/">CV</a>
            
          
            
            
            <a class="page-link" href="/publications/">Publications</a>
            
          
            
            
            <a class="page-link" href="/courses/">Courses</a>
            
          
            
            
            <a class="page-link" href="/software/">Software</a>
            
          
            
            
            <a class="page-link" href="/posts/">Posts</a>
            
          
        </div>
      </nav>
    
  </div>
</header>


    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <header class="post-header">
    <h1 class="post-title">Publications</h1>
  </header>

  <div class="post-content">
    <!--### Submitted Manuscripts-->

<h3 id="2017">2017</h3>

<ul>
  <li>
    <p>EL Jackson, <strong>SJ Spielman</strong>, and CO Wilke. 2017. <a href="http://biorxiv.org/content/early/2016/10/04/079061">Computational prediction of the tolerance to amino-acid deletion in green-fluorescent protein</a>. <em>PLOS ONE</em> 12(4): e0164905.
 <a href="https://github.com/wilkelab/eGFP_deletion_prediction" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/JacksonSpielmanWilke2016_PLOSONE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
  <li>
    <p>Z Kadlecova, <strong>SJ Spielman</strong>, D Loerke, A Mohanakrishnan, DK Reed, and SL Schmid. 2017. <a href="http://jcb.rupress.org/content/216/1/167">Regulation of clathrin-mediated endocytosis by hierarchical allosteric activation of AP2.</a> <em>J Cell Biol</em> 216(1): 167–179.
 <a href="/files/Kadlecovaetal2017_JCB.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></p>
  </li>
</ul>

<h3 id="2016">2016</h3>

<ul>
  <li>
    <p><strong>SJ Spielman</strong>, S Wan, and CO Wilke. 2016. <a href="http://www.genetics.org/content/early/2016/08/15/genetics.115.185264">A comparison of one-rate and two-rate inference frameworks for site-specific <em>dN/dS</em> estimation.</a> <em>Genetics</em> 204(2): 499–511.
 <a href="https://github.com/sjspielman/dnds_1rate_2rate" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/SpielmanWanWilke2016_Genetics.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
  <li>
    <p><strong>SJ Spielman</strong> and CO Wilke. 2016. <a href="http://mbe.oxfordjournals.org/content/early/2016/09/08/molbev.msw171">Extensively parameterized mutation–selection models reliably capture site-specific selective constraint.</a> <em>Mol Biol Evol</em> 33(11): 2990–3002.
 <a href="https://github.com/sjspielman/mutsel_benchmark" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/SpielmanWilke2016_MBE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
  <li>
    <p>EL Jackson, A Shahmoradi, <strong>SJ Spielman</strong>, BR Jack, and CO Wilke. 2016. <a href="http://onlinelibrary.wiley.com/doi/10.1002/pro.2920/full">Intermediate divergence levels maximize the strength of structure–sequence correlations in enzymes and viral proteins.</a> <em>Protein Sci</em> 25(7): 1341-1353.
 <a href="https://github.com/wilkelab/rate_variability_variation" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/Jacksonetal2016_ProtSci.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
  <li>
    <p>J Echave, <strong>SJ Spielman</strong>, and CO Wilke. 2016. <a href="http://dx.doi.org/10.1038/nrg.2015.18">Causes of evolutionary rate variation among protein sites.</a> <em>Nature Rev Genet</em> 17: 109-121.
 <a href="/files/EchaveSpielmanWilke2016_NRG.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></p>
  </li>
</ul>

<h3 id="2015">2015</h3>

<ul>
  <li>
    <p><strong>SJ Spielman</strong> and CO Wilke. 2015. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0139047">Pyvolve: A flexible Python module for simulating sequences along phylogenies.</a> <em>PLOS ONE</em> 10(9): e0139047.
 <a href="https://github.com/sjspielman/pyvolve" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/SpielmanWilke2015_PLoSONE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
  <li>
    <p>AG Meyer, <strong>SJ Spielman</strong>, T Bedford, and CO Wilke. 2015. <a href="http://ve.oxfordjournals.org/content/1/1/vev006">Time dependence of evolutionary metrics
during the 2009 pandemic influenza virus outbreak.</a>  <em>Virus Evolution.</em> 1(1):vev006-10.
 <a href="https://github.com/wilkelab/influenza_pH1N1_timecourse" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/Meyeretal2015_VE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
  <li>
    <p><strong>SJ Spielman</strong> and CO Wilke. 2015. <a href="http://mbe.oxfordjournals.org/content/32/4/1097">The relationship between <em>dN/dS</em> and scaled selection coefficients.</a> <em>Mol Biol Evol</em> 32(4): 1097-1108.
 <a href="http://github.com/clauswilke/Omega_Mutsel" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/SpielmanWilke2015_MBE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
  <li>
    <p><strong>SJ Spielman</strong>, K Kumar, and CO Wilke. <a href="http://dx.doi.org/10.7717/peerj.773">Comprehensive, structurally-curated alignment and phylogeny of vertebrate biogenic amine receptors.</a> <em>PeerJ</em> 3:e773. 2015.
 <a href="http://github.com/sjspielman/amine_receptors" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/SpielmanKumarWilke2015_PeerJ.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
</ul>

<h3 id="2014">2014</h3>

<ul>
  <li>
    <p><strong>SJ Spielman</strong>* , AG Meyer*, and CO Wilke. <a href="http://dx.doi.org/10.1101/011429">Increased evolutionary rate in the 2014 West African Ebola outbreak is due to transient polymorphism and not positive selection.</a> <em>bioRxiv</em>. 2014. *Authors contributed equally to this work.  <a href="https://github.com/wilkelab/EBOV_H1N1" class="info-link"><span class="fa fa-github" style="color:black"></span></a></p>
  </li>
  <li>
    <p><strong>SJ Spielman</strong>, ET Dawson, and CO Wilke. 2014. <a href="http://mbe.oxfordjournals.org/content/31/9/2496.long">Limited utility of residue masking for positive-selection inference.</a> <em>Mol Biol Evol</em> 31(9): 2496-2500. 2014.
 <a href="http://github.com/sjspielman/alignment_filtering" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/SpielmanDawsonWilke2014_MBE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
  <li>
    <p>A Shahmoradi, DK Sydykova, <strong>SJ Spielman</strong>, EL Jackson, ET Dawson, AG Meyer, and CO Wilke. 2014. <a href="http://dx.doi.org/10.1007/s00239-014-9644-x">Predicting evolutionary site variability from structure in viral proteins: buriedness, packing, flexibility, and design.</a> <em>J Mol Evol</em> 79:130–142. 2014.
 <a href="https://github.com/clauswilke/structural_prediction_of_ER" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/Shahmoradietal2014_JME.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
</ul>

<h3 id="2013">2013</h3>

<ul>
  <li>
    <p>MZ Tien, AG Meyer, DK Sydykova, <strong>SJ Spielman</strong>, and CO Wilke. 2013. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0080635">Maximum allowed solvent accessibilites of residues in proteins.</a> <em>PLOS One</em> 8(11):e80635. 2013.
 <a href="/files/Tienetal2013_PLoSONE.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></p>
  </li>
  <li>
    <p><strong>SJ Spielman</strong> and CO Wilke. 2013. <a href="http://link.springer.com/article/10.1007%2Fs00239-012-9538-8">Membrane environment imposes unique selection pressures in transmembrane domains of G-protein coupled receptors.</a> <em>J Mol Evol</em> 76(3):172-182. 2013.
 <a href="http://github.com/sjspielman/mammalian_gpcr_selection" class="info-link"><span class="fa fa-github" style="color:black">
 <a href="/files/SpielmanWilke2013_JME.pdf" class="info-link"><span class="fa fa-file-pdf-o" style="color:black"></span></a></span></a></p>
  </li>
</ul>

  </div>

</article>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <!--<h2 class="footer-heading">Home</h2>-->

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>Stephanie J. Spielman, PhD</li>
          <li>Institute for Genomics and Evolutionary Medicine</li>
          <li>Science and Education Research Center (SERC) 643</li>
          <li>Temple University</li>
          <li>1925 N 12th Street, Philadelphia, PA 19122</li>
          <li><a href="mailto:stephanie.spielman@temple.edu">stephanie.spielman@temple.edu</a></li>
        </ul>
      </div>

      <div class="footer-col footer-col-2">
      
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/sjspielman"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">sjspielman</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/stephspiel"><span class="icon icon--twitter"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">stephspiel</span></a>

          </li>
          
        </ul>
      </div>
<!--    
      <div class="footer-col footer-col-3">
        <p>Institute for Genomics and Evolutionary Medicine
</p>
      </div>
-->
    </div>

  </div>

</footer>


  </body>

</html>
