<!DOCTYPE html>
<html dir="ltr" lang="en">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>SJS</title>
	<meta name="keywords" content="SinglePaged, jekyll, template, github, blog, single page">
	<meta name="description" content="Stephanie's homepage">
  <link rel="stylesheet" href="combo.css">
  <link href='http://fonts.googleapis.com/css?family=Raleway:400,300,700' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" href="//netdna.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
  <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
	<link rel="apple-touch-icon" href="img/apple-touch-icon.png">
</head>
<body>
  <div id="main">

    <nav><ul>
      
        
        <li class="p-about"><a href="#about">about</a></li>
      
        
        <li class="p-publications"><a href="#publications">Publications</a></li>
      
        
        <li class="p-preprints"><a href="#preprints">Preprints</a></li>
      
        
        <li class="p-software"><a href="#software">Software</a></li>
      
        
        <li class="p-teaching"><a href="#teaching">Teaching</a></li>
      
    </ul></nav>


    
      
      <div id="about" class="section p-about">
        
        <div class="container center">
          <p><span class="fa-stack subtlecircle" style="font-size:100px; background:rgba(218, 229, 241, 0.5)">
  <i class="fa fa-circle-thin fa-stack-2x"></i>
  <i class="fa fa-flask fa-stack-1x"></i>
</span></p>

<h1 id="about-me">About Me</h1>

<p>I am a 5th year PhD Candidate in <a href="http://wilkelab.org">Claus Wilke’s lab</a> through the <a href="http://www.biosci.utexas.edu/graduate/eeb/">Ecology, Evolution, and Behavior graduate program</a> at UT Austin.
My CV (updated 11/25/15) is available for download <a href="../files/CV_SJSpielman.pdf">here</a>, and my research interests broadly encompass computational molecular evolution, phylogenetics, and protein evolution. 
My current research focuses on the relationships among different statistical models of coding-sequence evolution.</p>

        </div>
      </div>
    
      
      <div id="publications" class="section p-publications">
        
        <div class="subtlecircle sectiondivider faicon">
          <span class="fa-stack">
            <i class="fa fa-circle fa-stack-2x"></i>
            <i class="fa fa-newspaper-o fa-stack-1x"></i>
          </span>
          <h5 class="icon-title">Publications</h5>
        </div>
        
        <div class="container ">
          <h2 id="section">2015</h2>

<ul>
  <li>
    <p><strong>Spielman SJ</strong> and Wilke CO. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0139047">Pyvolve: A flexible Python module for simulating sequences along phylogenies.</a> <em>PLoS ONE.</em> 10(9): e0139047. 
 <a href="https://github.com/sjspielman/pyvolve" class="info-link"><span class="fa fa-github"> 
 <a href="/files/SpielmanWilke2015_PLoSONE.pdf" class="info-link"><span class="fa fa-file-pdf-o"></span></a></span></a></p>
  </li>
  <li>
    <p>Meyer AG, <strong>Spielman SJ</strong>,  Bedford T, and Wilke CO. <a href="http://ve.oxfordjournals.org/content/1/1/vev006">Time dependence of evolutionary metrics
during the 2009 pandemic influenza virus outbreak.</a>  <em>Virus Evolution.</em> 1(1):vev006-10.
 <a href="https://github.com/wilkelab/influenza_pH1N1_timecourse" class="info-link"><span class="fa fa-github">
 <a href="/files/Meyeretal2015_VE.pdf" class="info-link"><span class="fa fa-file-pdf-o"></span></a></span></a></p>
  </li>
  <li>
    <p><strong>Spielman SJ</strong> and Wilke CO. <a href="http://mbe.oxfordjournals.org/content/32/4/1097">The relationship between <em>dN/dS</em> and scaled selection coefficients.</a> <em>Mol Biol Evol</em> 32(4): 1097-1108.
 <a href="http://github.com/clauswilke/Omega_Mutsel" class="info-link"><span class="fa fa-github">
 <a href="/files/SpielmanWilke2015_MBE.pdf" class="info-link"><span class="fa fa-file-pdf-o"></span></a></span></a></p>
  </li>
  <li>
    <p><strong>Spielman SJ</strong>, Kumar K, and Wilke CO. <a href="http://dx.doi.org/10.7717/peerj.773">Comprehensive, structurally-curated alignment and phylogeny of vertebrate biogenic amine receptors.</a> <em>PeerJ</em> 3:e773. 2015.
 <a href="http://github.com/sjspielman/amine_receptors" class="info-link"><span class="fa fa-github">
 <a href="/files/SpielmanKumarWilke2015_PeerJ.pdf" class="info-link"><span class="fa fa-file-pdf-o"></span></a></span></a></p>
  </li>
</ul>

<p><br /></p>

<h2 id="section-1">2014</h2>

<ul>
  <li>
    <p><strong>Spielman SJ</strong>, Dawson ET, and Wilke CO. <a href="http://mbe.oxfordjournals.org/content/31/9/2496.long">Limited utility of residue masking for positive-selection inference.</a> <em>Mol Biol Evol</em> 31(9): 2496-2500. 2014.
 <a href="http://github.com/sjspielman/alignment_filtering" class="info-link"><span class="fa fa-github">
<a href="/files/SpielmanDawsonWilke2014_MBE.pdf" class="info-link"><span class="fa fa-file-pdf-o"></span></a></span></a></p>
  </li>
  <li>
    <p>Shahmoradi A, Sydykova DK, <strong>Spielman SJ</strong>, Jackson EL, Dawson ET, Meyer AG, Wilke CO. <a href="http://dx.doi.org/10.1007/s00239-014-9644-x">Predicting evolutionary site variability from structure in viral proteins: buriedness, packing, flexibility, and design.</a> <em>J Mol Evol</em> 79:130–142. 2014.
 <a href="https://github.com/clauswilke/structural_prediction_of_ER" class="info-link"><span class="fa fa-github">
 <a href="/files/Shahmoradietal2014_JME.pdf" class="info-link"><span class="fa fa-file-pdf-o"></span></a></span></a></p>
  </li>
</ul>

<p><br /></p>

<h2 id="section-2">2013</h2>

<ul>
  <li>
    <p>Tien MZ, Meyer AG, Sydykova DK, <strong>Spielman SJ</strong>, and Wilke CO. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0080635">Maximum allowed solvent accessibilites of residues in proteins.</a> <em>PLoS One</em> 8(11):e80635. 2013. 
 <a href="/files/Tienetal2013_PLoSONE.pdf" class="info-link"><span class="fa fa-file-pdf-o"></span></a></p>
  </li>
  <li>
    <p><strong>Spielman, SJ</strong> and Wilke, CO. <a href="http://link.springer.com/article/10.1007%2Fs00239-012-9538-8">Membrane environment imposes unique selection pressures in transmembrane domains of G-protein coupled receptors.</a> <em>J Mol Evol</em> 76(3):172-182. 2013.
 <a href="/files/SpielmanWilke2013_JME.pdf" class="info-link"><span class="fa fa-file-pdf-o">
  Erratum: <a href="/files/SpielmanWilke2013_JME_erratum.pdf" class="info-link"><span class="fa fa-file-pdf-o"></span></a></span></a></p>
  </li>
</ul>

        </div>
      </div>
    
      
      <div id="preprints" class="section p-preprints">
        
        <div class="subtlecircle sectiondivider faicon">
          <span class="fa-stack">
            <i class="fa fa-circle fa-stack-2x"></i>
            <i class="fa fa-folder-open-o fa-stack-1x"></i>
          </span>
          <h5 class="icon-title">Preprints</h5>
        </div>
        
        <div class="container ">
          <ul>
  <li>
    <p><strong>Spielman SJ</strong>, Wan S, and Wilke CO. <a href="http://dx.doi.org/10.1101/032805">One-rate models outperform two-rate models in site-specific dN/dS estimation.</a> <em>bioRxiv</em>. 2015.
 <a href="https://github.com/sjspielman/sitewise_dnds_mutsel" class="info-link"><span class="fa fa-github"></span></a></p>
  </li>
  <li>
    <p><strong>Spielman SJ</strong>* , Meyer AG*, and Wilke CO. <a href="http://dx.doi.org/10.1101/011429">Increased evolutionary rate in the 2014 West African Ebola outbreak is due to transient polymorphism and not positive selection.</a> <em>bioRxiv</em>. 2014. 
*Authors contributed equally to this work.  <a href="https://github.com/wilkelab/EBOV_H1N1" class="info-link"><span class="fa fa-github"></span></a></p>
  </li>
</ul>

        </div>
      </div>
    
      
      <div id="software" class="section p-software">
        
        <div class="subtlecircle sectiondivider faicon">
          <span class="fa-stack">
            <i class="fa fa-circle fa-stack-2x"></i>
            <i class="fa fa-desktop fa-stack-1x"></i>
          </span>
          <h5 class="icon-title">Software</h5>
        </div>
        
        <div class="container ">
          <ul>
  <li>
    <p><a href="https://sjspielman.github.io/pyvolve">Pyvolve</a>, a python platform to simulate sequences along a phylogeny using continuous-time Markov models.</p>
  </li>
  <li>
    <p><a href="https://sjspielman.github.io/alignfigR">alignfigR</a>, an R package to create multiple sequence alignment figures.</p>
  </li>
</ul>


        </div>
      </div>
    
      
      <div id="teaching" class="section p-teaching">
        
        <div class="subtlecircle sectiondivider faicon">
          <span class="fa-stack">
            <i class="fa fa-circle fa-stack-2x"></i>
            <i class="fa fa-pencil fa-stack-1x"></i>
          </span>
          <h5 class="icon-title">Teaching</h5>
        </div>
        
        <div class="container ">
          <p>I am deeply passionate about undergraduate (and beyond!) education, and I have been involved in teaching and designing a wide variety of courses. As a TA, I have taught Evolutionary Biology (both at UT Austin and Brown University), Biostatistics, and Computational Biology. Furthermore, during the summer of 2011, I worked for UT Austin’s Statistics and Data Sciences department (then the Statistics and Scientific Computing division) to re-design the undergraduate Biostatistics curriculum. In this capacity, I developed computer lab exerises and video tutorials for basic biostatistics analysis, ranging from summary statistics to multivariate linear modeling, in R.</p>

<p>Through UT Austin’s <a href="ccbb.biosci.utexas.edu/">Center for Computational Biology and Bioinformatics</a> (CCBB), I have been involved in multiple initiatives to teach biological computing and introductory programming to students of all levels (undergraduate, graduate, and postdoctoral). In particular, during the spring semester, I run, along with my co-instructor <a href="http://www.rebeccatarvin.com/">Rebecca Tarvin</a>, a peer-led working group on Biological Computing. Materials from our Spring 2015 group are freely available from <a href="http://sjspielman.org/UTbiocomputing2015">http://sjspielman.org/UTbiocomputing2015</a>.</p>

<p>During May 2015, I was the lead instructor for CCBB’s Introduction to Python course, as part of the Big Data in Biology Summer School. All course materials (which I designed) are freely available from <a href="http://sjspielman.org/ccbb_bigdata2015_python">http://sjspielman.org/ccbb_bigdata2015_python</a>.</p>

        </div>
      </div>
    


    <div id="footer" class="section text-white">
      <div class="container">
        
        
<p>Design by Tim O’Brien <a href="http://t413.com/">t413.com</a>
—
<a href="https://github.com/t413/SinglePaged">SinglePaged theme</a>
—
this site is <a href="https://github.com/t413/SinglePaged">open source</a></p>


      </div>
    </div>
  </div>


<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-49607422-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

</body>
<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
<script src="site.js"></script>
</html>
