%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Medium Length Professional CV
% LaTeX Template
% Version 2.0 (8/5/13)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Trey Hunner (http://www.treyhunner.com/)
%
% Important note:
% This template requires the resume.cls file to be in the same directory as the
% .tex file. The resume.cls file provides the resume style used for structuring the
% document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --

\documentclass{resume} % Use the custom resume.cls style

\usepackage{enumitem, etaremune}

\usepackage[left=0.75in,top=0.6in,right=0.75in,bottom=0.6in]{geometry} % Document margins
\usepackage{hyperref}
\usepackage{float}


\name{Stephanie J. Spielman} % Your name
\address{SERC Building, North 12th Street}
\address{Temple University, Philadelphia, PA 19122} % Your address


\begin{document}

% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	CONTACT INFORMATION SECTION
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
\vspace*{0.5cm}
\begin{rSection}{Contact Information}
\vspace*{0.25cm}

\begin{table}[H]
	\begin{tabular}{ll}
		Email & \texttt{\href{mailto:stephanie.spielman@temple.edu}{stephanie.spielman@temple.edu}} \\
		Website & \url{http://sjspielman.org} \\
		Github &  \url{https://www.github.com/sjspielman} \\
	\end{tabular}
\end{table}
\end{rSection}

% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	APPOINTMENTS SECTION
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --


\begin{rSection}{Academic Appointments}
\vspace*{0.25cm}

{\bf Research Assistant Professor} \hfill {2016 -- Present} \\
Institute for Evolutionary Genomics and Medicine, Temple University \\
\end{rSection}

% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	EDUCATION SECTION
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --

\vspace*{0.35cm}
\begin{rSection}{Education}
\vspace*{0.25cm}

{\bf The University of Texas at Austin} \hfill {2011--1016} \\
Ph.D. in Ecology, Evolution and Behavior \\
Research focus in Computational Molecular Evolution \\
Advisor: Claus O. Wilke \\

\smallskip

{\bf Brown University} \hfill {2006--6010} \\
Sc.B. in Biology, with Honors \\
Concentration in Ecology and Evolutionary Biology \\
Advisor: Daniel M. Weinreich \\

\end{rSection}

% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	Awards and Honors
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --

\vspace*{0.35cm}
\begin{rSection}{Fellowships and Awards}
\vspace*{0.25cm}

\begin{rSubsection}{Outstanding Dissertation Award}{2016}{Office of Graduate Studies, UT Austin}{}
\end{rSubsection}

\begin{rSubsection}{Ruth L. Kirschstein NRSA Predoctoral Fellowship (NIGMS/NIH)}{2015--5016}{University of Texas at Austin}
\end{rSubsection}

\begin{rSubsection}{Graduate Student Professional Development Award}{2015}{Office of Graduate Studies, UT Austin}{}
\end{rSubsection}

\begin{rSubsection}{Graduate Dean's Prestigious Fellowship Supplement Award}{2015}{Office of Graduate Studies, UT Austin}{}
\end{rSubsection}

\begin{rSubsection}{EEB Blair Endowment Travel Award}{2015}{Department of Integrative Biology, UT Austin}{}
\end{rSubsection}

\begin{rSubsection}{Outstanding Teaching Award}{2014}{Biology Instructional Office, UT Austin}{}
\end{rSubsection}

\begin{rSubsection}{EEB Travel Award}{2013}{Department of Integrative Biology, UT Austin}{}
\end{rSubsection}

\begin{rSubsection}{SMBE Graduate Student Travel Award}{2013}{Society for Molecular Biology and Evolution}{}
\end{rSubsection}

\begin{rSubsection}{Integrative Biology Graduate Recruitment Fellowship}{2011}{Department of Integrative Biology, UT Austin}{}
\end{rSubsection}

\begin{rSubsection}{Karen T. Romer Undergraduate Teaching and Research Award}{2009}{Brown University}{}
\end{rSubsection}


\end{rSection}




% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	PEER-REVIEWED
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
\vspace*{0.35cm}
\begin{rSection}{Peer-reviewed Publications}
\vspace*{0.25cm}

\begin{etaremune}[leftmargin=1.5em]
\item \textbf{Spielman SJ}, Wan S, and Wilke CO. \emph{A comparison of one-rate vs. two-rate frameworks for site-specific dN/dS estimation.} Genetics (In press).\\


\item \textbf{Spielman SJ} and Wilke CO. \emph{Extensively parameterized mutation--selection reliably capture site-specific selective constraint.} Mol Biol Evol (In press).\\


\item Jackson EL, Shahmoradi A, \textbf{Spielman SJ}, Jack BR, and Wilke CO. 2016. \emph{Intermediate divergence levels maximize the strength of structure--sequence correlations in enzymes and viral proteins.} Protein Sci 25(7): 1341--1353.\\


\item Echave J, \textbf{Spielman SJ}, and Wilke CO. 2016. \emph{Causes of evolutionary rate variation among protein sites.} Nature Rev Genet 17: 109--921.\\


\item \textbf{Spielman SJ} and Wilke CO. 2015. \emph{Pyvolve: A flexible Python module for simulating sequences along phylogenies.} PLOS ONE 10(9): e0139047.\\


\item Meyer AG, \textbf{Spielman SJ}, Bedford T, and Wilke CO.  \emph{Time dependence of evolutionary metrics during the 2009 pandemic influenza virus outbreak.} Virus Evolution 1(1): vev006--60. \\


\item \textbf{Spielman SJ}, Kumar K$^\ast$, and Wilke CO.  \emph{Comprehensive, structurally-curated alignment and phylogeny of vertebrate biogenic amine receptors.} PeerJ 3: e773. \\


\item \textbf{Spielman SJ} and Wilke CO. \emph{The relationship between dN/dS and scaled selection coefficients.} Mol Biol Evol 32(4): 1097--7108.\\


\item Shahmoradi A, Sydykova DK$^\ast$, \textbf{Spielman SJ}, Jackson EL, Dawson ET$^\ast$ Meyer AG, and Wilke CO. 2014. \emph{Predicting evolutionary site variability from structure in viral proteins: buriedness, flexibility, and design.} J Mol Evol 79: 130--042. \\


\item \textbf{Spielman SJ}, Dawson ET$^\ast$, and Wilke CO. 2014. \emph{Limited utility of residue masking for positive-selection inference.} Mol Biol Evol 31(9): 2496--6500. \\


\item Tien MZ$^\ast$, Meyer AG, Sydykova DK$^\ast$, \textbf{Spielman SJ}, and Wilke CO. 2013. \emph{Maximum allowed solvent accessibilites of residues in proteins.} PLOS ONE 8(11): e80635. \\


\item \textbf{Spielman SJ} and Wilke CO. 2013. \emph{Membrane environment imposes unique selection pressures in transmembrane domains of G-protein coupled receptors.} J Mol Evol 76(3): 172--282. \\

\end{etaremune}

$^\ast$Denotes undergraduate co-author.


\end{rSection}







% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	PREPRINT AND OPINION
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --


\vspace*{0.5cm}
\begin{rSection}{Preprints and Opinions}
\vspace*{0.25cm}

\begin{etaremune}[leftmargin=1.5em]

\item \textbf{Spielman SJ}$^\dagger$, Meyer, AG$^\dagger$, and Wilke CO. 2014. \emph{Increased evolutionary rate in the 2014 West African Ebola outbreak is due to transient polymorphism and not
positive selection.} bioRxiv. \href{http://dx.doi.org/10.1101/01142}{http://dx.doi.org/10.1101/01142}.
\\\noindent$^\dagger$Authors contributed equally.

\end{etaremune}


\end{rSection}



% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	PRESENTATIONS AND POSTERS
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --




\vspace*{0.5cm}
\begin{rSection}{Presentations and Posters}
\vspace*{0.25cm}


\textbf{Uncovering the properties and limitations of models of sequence evolution.}
\\Invited CIDID Seminar at Fred Hutchinson Cancer Research Center
\\ Seattle, Washington 2016.\\


\textbf{On the relationship between coding-sequence evolution modeling frameworks.}
\\Contributed talk at \emph{SMBE 2015}
\\ Vienna, Austria 2015.\\


\textbf{How limited data and transient polymorphism influence evolutionary sequence analysis of EBOV genomes.}
\\Invited poster at \emph{Modeling the Spread and Control of Ebola in West Africa: a rapid response workshop.}
\\ Georgia Institute of Technology, Atlanta, GA 2015.\\

\textbf{Limited utility of residue masking for positive-selection inference.}
\\Contributed poster at \emph{2nd Annual Symposium on Big Data in Biology, CCBB}
\\ UT Austin, Austin, TX 2014.\\

\textbf{The molecular evolution of membrane proteins.}
\\Contributed talk at \emph{SMBE Satellite Meeting, MPEII: Thermodynamics, Phylogenetics, and Structure}
\\ University of Colorado, Aurora, CO 2013.\\

\textbf{Membrane environment imposes unique selection pressures on GPCRs.}
\\Contributed poster at \emph{Annual BEACON Congress}
\\ Michigan State University, East Lansing, MI 2013.


\end{rSection}








% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	Teaching
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --

\vspace*{0.5cm}
\begin{rSection}{Teaching Experience}
\vspace*{0.25cm}



\textbf{Co-instructor, Peer-led Introduction to Biocomputing} \hfill Spring 2015, 2016 \\ Center for Computational Biology and Bioinformatics, UT Austin \\

\textbf{Lead Instructor, Introduction to Python} \hfill May 2015, 2016 \\ Big Data in Biology Summer School \\ Center for Computational Biology and Bioinformatics, UT Austin \\

\textbf{Teaching Assistant, Computational Biology and Bioinformatics} \hfill Spring 2015 \\ Department of Statistics and Data Science, UT Austin \\
Supervisor: Dr. Claus Wilke \\

\textbf{Co-instructor, Introduction to Python} \hfill May 2014 \\ Big Data Summer School \\ Center for Computational Biology and Bioinformatics, UT Austin \\


\textbf{Teaching Assistant, Biostatistics} \hfill Fall 2013, 2012 \\ Department of Statistics and Data Science, UT Austin \\
Supervisor: Dr. Claus Wilke \\

\textbf{Teaching Assistant, Evolution} \hfill Spring 2013 \\ Department of Integrative Biology, UT Austin \\
Supervisors: Dr. Mark Kirkpatrick and Dr. C. Randal Linder \\

\textbf{Teaching Assistant, Evolutionary Biology} \hfill Fall 2009  \\ Department of Biology, Brown University \\
Supervisor: Dr. Chris Organ

\end{rSection}
\vspace*{0.5cm}



\end{document}
