<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Brief Introduction to Pyvolve</title>
  <meta name="description" content="Brief introduction to Pyvolve">

  <link rel="stylesheet" href="/assets/main.css">
  <link rel="canonical" href="http://localhost:4000/2015/06/01/pyvolve-tutorial.html">
  <link rel="alternate" type="application/rss+xml" title="Home" href="/feed.xml">
  <link rel="stylesheet" href="/assets/prism.css">
  <script src="https://use.fontawesome.com/a672cf20b0.js"></script>
  
</head>


  <body>
    <script src="/assets/prism.js"></script>
    <header class="site-header" role="banner">

  <div class="wrapper-header">
    
    
    <a class="site-title" href="/">Home</a>
  
    
      <nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger">
          
            
            
            <a class="page-link" href="/cv/">CV</a>
            
          
            
            
            <a class="page-link" href="/publications/">Publications</a>
            
          
            
            
            <a class="page-link" href="/courses/">Courses</a>
            
          
            
            
            <a class="page-link" href="/software/">Software</a>
            
          
            
            
            <a class="page-link" href="/posts/">Posts</a>
            
          
        </div>
      </nav>
    
  </div>
</header>


    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title" itemprop="name headline">Brief Introduction to Pyvolve</h1>
    <p class="post-meta">
      <time datetime="2015-06-01T00:00:00-04:00" itemprop="datePublished">
        
        Jun 1, 2015
      </time>
      </p>
  </header>

  <div class="post-content" itemprop="articleBody">
    <h1 id="brief-introduction-to-pyvolve">Brief introduction to Pyvolve</h1>

<p>This blog post contains a brief tutorial for <a href="http://sjspielman.org/pyvolve">Pyvolve</a>, an open-source Python module for simulating sequences along phylogenies (<a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0139047">Spielman and Wilke, 2015</a>). A comprehensive user-manual for Pyvolve is available with the most recent release from this <a href="https://github.com/sjspielman/pyvolve">github repository</a>.</p>

<p>Pyvolve is written in pure Python, with dependencies of NumPy, SciPy, and Biopython. The Pyvolve framework is extremely flexible, allowing you to simulate sequences according to virtually all standard models of nucleotide, amino acid, and codon data, and you can customize <em>all</em> model parameters to your heart’s content. Further, Pyvolve allows you to provide a custom rate matrix, if the available models are not quite what you’re looking for (however, please feel free to get in touch with me if you would like to request that a new model be included!).</p>

<p>Pyvolve incorporates both site and temporal heterogeneity, and, as you’ll see in the preprint linked above, contains several novel simulation features. Below, I show some simple examples of Pyvolve simulations. In general, sequence simulations require several you to do a few things:</p>

<ul>
  <li>Specify a phylogeny (with branch lengths!)</li>
  <li>Define any evolutionary model(s) to use. In Pyvolve, these are <strong>Model</strong> objects.</li>
  <li>Assign model(s) to partition(s). In Pyvolve, these are <strong>Partition</strong> objects.</li>
  <li>Evolve, using the callable <strong>Evolver</strong> class.</li>
</ul>

<p>Partitions are essentially a convenient way of defining “domains” – each partition can evolve according to a distinct evolutionary model (provided that all partitions evolve the same state, e.g. nucleotides, amino acids, or codons), and each partition can have differing degrees of heterogeneity.</p>

<p>Examples shown below are minimal and do not capture the full power of Pyvolve – to really see what Pyvolve can do, have a look at the <a href="https://github.com/sjspielman/pyvolve/raw/master/user_manual/pyvolve_manual.pdf">user manual</a>!</p>

<h2 id="simulating-nucleotide-sequences">Simulating nucleotide sequences</h2>

<p>This simple example demonstrates how to evolve nucleotide sequences.</p>

<figure class="highlight"><pre><code class="language-python" data-lang="python"><span class="kn">import</span> <span class="nn">pyvolve</span>

<span class="c"># Define a phylogeny, from a file containing a newick tree</span>
<span class="n">my_tree</span> <span class="o">=</span> <span class="n">pyvolve</span><span class="o">.</span><span class="n">read_tree</span><span class="p">(</span><span class="nb">file</span> <span class="o">=</span> <span class="s">"file_with_tree.tre"</span><span class="p">)</span>

<span class="c"># Define a nucleotide model, as a Model object.</span>
<span class="n">my_model</span> <span class="o">=</span> <span class="n">Model</span><span class="p">(</span><span class="s">"nucleotide"</span><span class="p">)</span>

<span class="c"># Assign the model to a Partition. The size argument indicates to evolve 250 positions</span>
<span class="n">my_partition</span> <span class="o">=</span> <span class="n">Partition</span><span class="p">(</span><span class="n">models</span> <span class="o">=</span> <span class="n">my_model</span><span class="p">,</span> <span class="n">size</span> <span class="o">=</span> <span class="mi">250</span><span class="p">)</span>

<span class="c"># Evolve!</span>
<span class="n">my_evolver</span> <span class="o">=</span> <span class="n">Evolver</span><span class="p">(</span><span class="n">partitions</span> <span class="o">=</span> <span class="n">my_partition</span><span class="p">,</span> <span class="n">tree</span> <span class="o">=</span> <span class="n">my_tree</span><span class="p">)</span>
<span class="n">my_evolver</span><span class="p">()</span></code></pre></figure>

<p><br />
The code shown above will simulate a nucleotide alignment of 250 positions along the phylogeny provided in <code class="highlighter-rouge">file_with_tree.tre</code>. This code simulates nucleotides according to default parameters: mutation rates among nucleotides are equal, and nucleotide equilibrium frequencies are equal at 0.25 each. We can customize these parameters by adding a <em>second argument to Model</em>: a dictionary of parameters to customize.</p>

<p>To customize mutation rates, we can use the key <code class="highlighter-rouge">"mu"</code>. This key should have an associated value of a dictionary of mutation rates. Mutation rates are symmetric, denotated by keys <code class="highlighter-rouge">"AT"</code>, <code class="highlighter-rouge">"AC"</code>, etc. (where “AT” is the rate from A to T, and conversely T to A). To customize frequencies, we can use the key <code class="highlighter-rouge">"state_freqs"</code>, whose associated value should be a list/numpy array of frequencies ordered ACGT.</p>

<p>This code chunk simulates nucleotide sequences with customized parameters:</p>

<figure class="highlight"><pre><code class="language-python" data-lang="python"><span class="kn">import</span> <span class="nn">pyvolve</span>

<span class="c"># Define a phylogeny, from a file containing a newick tree</span>
<span class="n">my_tree</span> <span class="o">=</span> <span class="n">pyvolve</span><span class="o">.</span><span class="n">read_tree</span><span class="p">(</span><span class="nb">file</span> <span class="o">=</span> <span class="s">"file_with_tree.tre"</span><span class="p">)</span>

<span class="c"># Define a nucleotide model with custom parameters!</span>
<span class="n">mutation</span> <span class="o">=</span> <span class="p">{</span><span class="s">"AC"</span><span class="p">:</span> <span class="mf">1.5</span><span class="p">,</span> <span class="s">"AG"</span><span class="p">:</span> <span class="mf">2.5</span><span class="p">,</span> <span class="s">"AT"</span><span class="p">:</span> <span class="mf">0.5</span><span class="p">,</span> <span class="s">"CG"</span><span class="p">:</span> <span class="mf">0.8</span><span class="p">,</span> <span class="s">"CT"</span><span class="p">:</span> <span class="mf">0.99</span><span class="p">,</span> <span class="s">"GT"</span><span class="p">:</span> <span class="mf">1.56</span><span class="p">}</span>
<span class="n">frequencies</span> <span class="o">=</span> <span class="p">[</span><span class="mf">0.25</span><span class="p">,</span> <span class="mf">0.3</span><span class="p">,</span> <span class="mf">0.1</span><span class="p">,</span> <span class="mf">0.35</span><span class="p">]</span> <span class="c"># f(A) = 0.25, f(C) = 0.3, etc.</span>
<span class="n">my_model</span> <span class="o">=</span> <span class="n">Model</span><span class="p">(</span><span class="s">"nucleotide"</span><span class="p">,</span> <span class="p">{</span><span class="s">"mu"</span><span class="p">:</span> <span class="n">mutation</span><span class="p">,</span> <span class="s">"state_freqs"</span><span class="p">:</span> <span class="n">frequencies</span><span class="p">}</span> <span class="p">)</span>

<span class="c"># Assign the model to a Partition.</span>
<span class="n">my_partition</span> <span class="o">=</span> <span class="n">Partition</span><span class="p">(</span><span class="n">models</span> <span class="o">=</span> <span class="n">my_model</span><span class="p">,</span> <span class="n">size</span> <span class="o">=</span> <span class="mi">250</span><span class="p">)</span>

<span class="c"># Evolve!</span>
<span class="n">my_evolver</span> <span class="o">=</span> <span class="n">Evolver</span><span class="p">(</span><span class="n">partitions</span> <span class="o">=</span> <span class="n">my_partition</span><span class="p">,</span> <span class="n">tree</span> <span class="o">=</span> <span class="n">my_tree</span><span class="p">)</span>
<span class="n">my_evolver</span><span class="p">()</span></code></pre></figure>

<p>For those of you who, like myself, tend towards some minor, completely socially-acceptable laziness, you can alternatively specify mutation rates with simply the key <code class="highlighter-rouge">"kappa"</code>, which represents the transition-to-transversion bias. Here’s how to define such a model:</p>

<figure class="highlight"><pre><code class="language-python" data-lang="python"><span class="c"># Define a nucleotide model kappa</span>
<span class="n">frequencies</span> <span class="o">=</span> <span class="p">[</span><span class="mf">0.25</span><span class="p">,</span> <span class="mf">0.3</span><span class="p">,</span> <span class="mf">0.1</span><span class="p">,</span> <span class="mf">0.35</span><span class="p">]</span> <span class="c"># f(A) = 0.25, f(C) = 0.3, etc.</span>
<span class="n">my_model</span> <span class="o">=</span> <span class="n">Model</span><span class="p">(</span><span class="s">"nucleotide"</span><span class="p">,</span> <span class="p">{</span><span class="s">"kappa"</span><span class="p">:</span> <span class="mf">3.25</span><span class="p">,</span> <span class="s">"state_freqs"</span><span class="p">:</span> <span class="n">frequencies</span><span class="p">}</span> <span class="p">)</span></code></pre></figure>

<p><br />
Now we’re cookin’! Let’s add some more bells and whistles, like <em>rate heterogeneity</em>. In the simulations shown above, all positions evolve according to exactly the same model and the same rate. We can incorporate rate heterogeneity by adding a few <em>keyword arguments</em> when defining our Model object. In this example, we will specify rate heterogeneity with a custom distribution, although as you’ll see in the <a href="https://github.com/sjspielman/pyvolve/raw/master/user_manual/pyvolve_manual.pdf">user manual</a>, you can also specify that rates be distribution according to a gamma distribution.</p>

<p>To implement rate heterogeneity (this holds for nucleotide and amino-acid models!), you need to specify the <em>scalar factors</em> which govern the heterogeneity, and a list of <em>probabilities</em> associated with each factor. This list will determine the probability that a given site evolves according to the associated factor. (Note that if you don’t specify these probabilties, each category will be equally likely).</p>

<p>Let’s go ahead and add in four rate categories, some slow and some fast, with associated probabilities. Specify a list of rate factors with the argument <code class="highlighter-rouge">rate_factors</code>, and specify a list of probabilities with the argument <code class="highlighter-rouge">rate_probs</code> (should sum to 1!). These lists are associated 1:1, as in the first item in <code class="highlighter-rouge">rate_factors</code> will have a probability equal to the first item in <code class="highlighter-rouge">rate_probs</code>.</p>

<figure class="highlight"><pre><code class="language-python" data-lang="python"><span class="kn">import</span> <span class="nn">pyvolve</span>

<span class="c"># Define a phylogeny, from a file containing a newick tree</span>
<span class="n">my_tree</span> <span class="o">=</span> <span class="n">pyvolve</span><span class="o">.</span><span class="n">read_tree</span><span class="p">(</span><span class="nb">file</span> <span class="o">=</span> <span class="s">"file_with_tree.tre"</span><span class="p">)</span>

<span class="c"># Define our parameters</span>
<span class="n">mutation</span> <span class="o">=</span> <span class="p">{</span><span class="s">"AC"</span><span class="p">:</span> <span class="mf">1.5</span><span class="p">,</span> <span class="s">"AG"</span><span class="p">:</span> <span class="mf">2.5</span><span class="p">,</span> <span class="s">"AT"</span><span class="p">:</span> <span class="mf">0.5</span><span class="p">,</span> <span class="s">"CG"</span><span class="p">:</span> <span class="mf">0.8</span><span class="p">,</span> <span class="s">"CT"</span><span class="p">:</span> <span class="mf">0.99</span><span class="p">,</span> <span class="s">"GT"</span><span class="p">:</span> <span class="mf">1.56</span><span class="p">}</span>
<span class="n">frequencies</span> <span class="o">=</span> <span class="p">[</span><span class="mf">0.25</span><span class="p">,</span> <span class="mf">0.3</span><span class="p">,</span> <span class="mf">0.1</span><span class="p">,</span> <span class="mf">0.35</span><span class="p">]</span> <span class="c"># f(A) = 0.25, f(C) = 0.3, etc.</span>

<span class="n">factors</span> <span class="o">=</span> <span class="p">[</span><span class="mf">3.5</span><span class="p">,</span> <span class="mf">2.5</span><span class="p">,</span> <span class="mf">0.08</span><span class="p">,</span> <span class="mf">0.005</span><span class="p">]</span> <span class="c"># Two fast categories, and two slow categories</span>
<span class="n">probs</span>   <span class="o">=</span> <span class="p">[</span><span class="mf">0.05</span><span class="p">,</span> <span class="mf">0.1</span><span class="p">,</span> <span class="mf">0.5</span><span class="p">,</span> <span class="mf">0.35</span><span class="p">]</span>  <span class="c"># The fast rates will occur with relatively low probabilities</span>

<span class="c"># Define our model with all parameters</span>
<span class="n">my_model</span> <span class="o">=</span> <span class="n">Model</span><span class="p">(</span><span class="s">"nucleotide"</span><span class="p">,</span> <span class="p">{</span><span class="s">"mu"</span><span class="p">:</span> <span class="n">mutation</span><span class="p">,</span> <span class="s">"state_freqs"</span><span class="p">:</span> <span class="n">frequencies</span><span class="p">},</span> <span class="n">rate_factors</span> <span class="o">=</span> <span class="n">factors</span><span class="p">,</span> <span class="n">rate_probs</span> <span class="o">=</span> <span class="n">probs</span><span class="p">)</span>

<span class="c"># Assign the model to a Partition.</span>
<span class="n">my_partition</span> <span class="o">=</span> <span class="n">Partition</span><span class="p">(</span><span class="n">models</span> <span class="o">=</span> <span class="n">my_model</span><span class="p">,</span> <span class="n">size</span> <span class="o">=</span> <span class="mi">250</span><span class="p">)</span>

<span class="c"># Evolve!</span>
<span class="n">my_evolver</span> <span class="o">=</span> <span class="n">Evolver</span><span class="p">(</span><span class="n">partitions</span> <span class="o">=</span> <span class="n">my_partition</span><span class="p">,</span> <span class="n">tree</span> <span class="o">=</span> <span class="n">my_tree</span><span class="p">)</span>
<span class="n">my_evolver</span><span class="p">()</span></code></pre></figure>

<p>By default, Pyvolve outputs several files:</p>

<ul>
  <li>simulated_alignment.fasta</li>
  <li>site_rates.txt</li>
  <li>site_rates_info.txt</li>
</ul>

<p>The first file contains the simulated alignment, and the latter two files contain information about site-specific rates and/or parameters. Using these two files, you can determine at which rate each site evolved. Note that you can suppress the creation of or change the name of these files with certain arguments when <em>calling</em> your Evolver object – see the user manual!</p>

<p><br /> And finally, one more example - what if we wanted to use <em>multiple models</em> in our simulation? For this task, we’ll need to define multiple Partition objects. In the example below, one Partition object will be assigned default parameters, and one Partition will be assigned custom parameters.</p>

<figure class="highlight"><pre><code class="language-python" data-lang="python"><span class="kn">import</span> <span class="nn">pyvolve</span>

<span class="c"># Define a phylogeny, from a file containing a newick tree</span>
<span class="n">my_tree</span> <span class="o">=</span> <span class="n">pyvolve</span><span class="o">.</span><span class="n">read_tree</span><span class="p">(</span><span class="nb">file</span> <span class="o">=</span> <span class="s">"file_with_tree.tre"</span><span class="p">)</span>

<span class="c"># Define default model</span>
<span class="n">model1</span> <span class="o">=</span> <span class="n">Model</span><span class="p">(</span><span class="s">"nucleotide"</span><span class="p">)</span>

<span class="c"># Define customized model (notice, no site heterogeneity this time!)</span>
<span class="n">mutation</span> <span class="o">=</span> <span class="p">{</span><span class="s">"AC"</span><span class="p">:</span> <span class="mf">1.5</span><span class="p">,</span> <span class="s">"AG"</span><span class="p">:</span> <span class="mf">2.5</span><span class="p">,</span> <span class="s">"AT"</span><span class="p">:</span> <span class="mf">0.5</span><span class="p">,</span> <span class="s">"CG"</span><span class="p">:</span> <span class="mf">0.8</span><span class="p">,</span> <span class="s">"CT"</span><span class="p">:</span> <span class="mf">0.99</span><span class="p">,</span> <span class="s">"GT"</span><span class="p">:</span> <span class="mf">1.56</span><span class="p">}</span>
<span class="n">frequencies</span> <span class="o">=</span> <span class="p">[</span><span class="mf">0.25</span><span class="p">,</span> <span class="mf">0.3</span><span class="p">,</span> <span class="mf">0.1</span><span class="p">,</span> <span class="mf">0.35</span><span class="p">]</span> <span class="c"># f(A) = 0.25, f(C) = 0.3, etc.</span>
<span class="n">model2</span> <span class="o">=</span> <span class="n">Model</span><span class="p">(</span><span class="s">"nucleotide"</span><span class="p">,</span> <span class="p">{</span><span class="s">"mu"</span><span class="p">:</span> <span class="n">mutation</span><span class="p">,</span> <span class="s">"state_freqs"</span><span class="p">:</span> <span class="n">frequencies</span><span class="p">})</span>

<span class="c"># Assign each model to a Partition.</span>
<span class="n">partition1</span> <span class="o">=</span> <span class="n">Partition</span><span class="p">(</span><span class="n">models</span> <span class="o">=</span> <span class="n">model1</span><span class="p">,</span> <span class="n">size</span> <span class="o">=</span> <span class="mi">100</span><span class="p">)</span>
<span class="n">partition2</span> <span class="o">=</span> <span class="n">Partition</span><span class="p">(</span><span class="n">models</span> <span class="o">=</span> <span class="n">model2</span><span class="p">,</span> <span class="n">size</span> <span class="o">=</span> <span class="mi">200</span><span class="p">)</span>

<span class="c"># Evolve by providing both partitions in a list to Evolver</span>
<span class="n">my_evolver</span> <span class="o">=</span> <span class="n">Evolver</span><span class="p">(</span><span class="n">partitions</span> <span class="o">=</span> <span class="p">[</span><span class="n">partition1</span><span class="p">,</span> <span class="n">partition2</span><span class="p">],</span> <span class="n">tree</span> <span class="o">=</span> <span class="n">my_tree</span><span class="p">)</span>
<span class="n">my_evolver</span><span class="p">()</span></code></pre></figure>

<p>In the resulting sequence file, the first 100 positions will have evolved according to <code class="highlighter-rouge">model1</code>, and the next 200 positions (there will be a total of 300 positions!) will evolve according to <code class="highlighter-rouge">model2</code>.</p>

<p>For more, yes more!, ways to use Pyvolve, check out the (drumroll…) <a href="https://github.com/sjspielman/pyvolve/raw/master/user_manual/pyvolve_manual.pdf">user manual</a>! Please feel free to post any questions and/or file bug reports on Pyvolve’s github repository Issues section. Enjoy!</p>

<p><br /><br /><br /></p>

  </div>

  
</article>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <!--<h2 class="footer-heading">Home</h2>-->

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>Stephanie J. Spielman, PhD</li>
          <li>Institute for Genomics and Evolutionary Medicine</li>
          <li>Science and Education Research Center (SERC) 643</li>
          <li>Temple University</li>
          <li>1925 N 12th Street, Philadelphia, PA 19122</li>
          <li><a href="mailto:stephanie.spielman@temple.edu">stephanie.spielman@temple.edu</a></li>
        </ul>
      </div>

      <div class="footer-col footer-col-2">
      
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/sjspielman"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">sjspielman</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/stephspiel"><span class="icon icon--twitter"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">stephspiel</span></a>

          </li>
          
        </ul>
      </div>
<!--    
      <div class="footer-col footer-col-3">
        <p>Institute for Genomics and Evolutionary Medicine
</p>
      </div>
-->
    </div>

  </div>

</footer>


  </body>

</html>
